_data/references.yml

```diff
@@ -0,0 +1,10 @@
+- number: 1
+  title: "Automated tagging of LATEX documents — what is possible today, in 2023? TUGboat, 44(2):262–266, 2023"
+  authors: U. Fischer and F. Mittelbach
+  url: "https://www.latex-project.org/publications/2023-UFi-FMi-TUG-tb137fischer-tagging23.pdf"
+
+- number: 2
+  title: "Enhancing LATEX to automatically produce tagged and accessible PDF. TUGboat, 45(1), 2024"
```

Comment:
I had fixed LATEX -> LaTeX in my version

Reply:
fixed in f969da8